apidoc/Titanium/UI/ScrollView.yml

```diff
@@ -114,9 +114,9 @@ events:
       - name: contentSize
         summary: |
             The current content size of the scroll view defined by its `width` and `height` properties.
-            Available in Titanium SDK 5.2.0 and later.
         type: Dictionary
-        platforms: [iphone, ipad]
+        platforms: [android, iphone, ipad]
+        since: { iphone: "5.2.0", ipad: "5.2.0", android: "8.2.0" }
```

Comment:
Titanium 8.2.0 is planned to be an iOS 13 only release.
No Android or Windows changes are allowed.

Would you mind changing the version from 8.2.0 to 8.3.0 please?